commanderbot_ext/ping/ping_cog.py

```diff
@@ -0,0 +1,13 @@
+import discord
+from commanderbot_lib.logging import Logger, get_clogger
+from discord.ext.commands import Bot, Cog, Context, command
+
+
+class PingCog(Cog):
+    def __init__(self, bot: Bot):
+        self.bot: Bot = bot
+        self._log: Logger = get_clogger(self)
+
+    @command(name="ping")
+    async def cmd_ping(self, ctx: discord.Message):
```

Comment:
`ctx` isn't a `Message`, it's an instance of `Context` from `discord.ext.commands`.

Reply:
Fixed in latest commit